deployment/helm/node-feature-discovery/templates/topology-gc.yaml

```
@@ -0,0 +1,64 @@
+{{- if and ( eq .Values.topologyGC.enable true ) ( eq .Values.topologyUpdater.enable true ) -}}
```

nit: could be simplified
```suggestion
{{- if and .Values.topologyGC.enable .Values.topologyUpdater.enable -}}
```

---

done